.github/workflows/ci_pullrequest.yml

```diff
@@ -278,3 +278,22 @@ jobs:
       - "pylint"
       - "tests-postgres"
       - "tests-mysql"
+
+  changelog:
+    if: github.base_ref == 'develop' || github.base_ref == 'next'
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: "Check for changelog entry"
+        uses: brettcannon/check-for-changed-files@v1
+        with:
+          file-pattern: |
+            changes/*.added
+            changes/*.changed
+            changes/*.fixed
+            changes/*.removed
+            nautobot/docs/release-notes/version-*.*.md
+          failure-message: >
+            Change log fragment missing. Please add a changelog fragment to ./changes in the
+            branch ${{github.head_ref}}. See
+            https://nautobot.readthedocs.io/en/stable/development/#creating-changelog-fragments
+            for more information on creating changelog fragments.
```

Comment:
```suggestion
      - name: "Setup environment"
        uses: "networktocode/gh-action-setup-poetry-environment@v2"
      - name: "Check for changelog entry"
        run: "poetry run towncrier check --compare-with origin/${{ github.base_ref }}"
```

🤷

Reply:
This increases the runtime by about 30 seconds but it works.